e2e/cypress/support/api/user.js

```diff
@@ -64,7 +64,7 @@ Cypress.Commands.add('apiLogout', () => {
     });
 
     // * Verify logged out
-    cy.visit('/login?extra=expired').url().should('include', '/login?extra=expired');
+    cy.visit('/login?extra=expired').url().should('include', '/login');
```

Comment:
At times after user is logged out, the user is redirected to `/login` page instead of `/login?extra=expired` page. Hence this change was necessary to avoid false failures